File: OldProject/SpaceFist/SpaceFist/Components/Abstract/Component.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.Xna.Framework;

namespace SpaceFist.Components.Abstract
{
    /// <summary>
    /// The base interface of any more specific component interface.
    /// 
    /// All components directly or indirectly implement this interface.
    /// </summary>
    public interface Component
    {
        void Update(GameData gameData, Entity obj);
    }
}

File: core/src/com/spacefist/components/abst/Component.java
package com.spacefist.components.abst;

import com.spacefist.GameData;
import com.spacefist.entities.Entity;

    /**
     * The base interface of any more specific component interface.
     *
     * All components directly or indirectly implement this interface.
     */
    public interface Component
    {
        void update(GameData gameData, Entity obj);
    }
